actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -94,8 +94,6 @@ def self.add(key, &block)
 
     # This method is the opposite of add method.
     #
-    # Usage:
```

Comment:
For instance, look above at line 79. The example is introduced by:

    To use the csv renderer in a controller action: